src/table/table.component.ts

```diff
@@ -173,74 +173,76 @@ import { TableRowSize } from "./table.types";
-			*ngIf="!noData; else noDataTemplate">
-		</tbody>
-		<ng-template #noDataTemplate><ng-content></ng-content></ng-template>
-		<tfoot>
-			<ng-template
-				[ngTemplateOutlet]="footerTemplate">
-			</ng-template>
-			<tr *ngIf="this.model.isLoading">
-				<td class="table_loading-indicator">
-					<div class="bx--loading bx--loading--small">
-						<svg class="bx--loading__svg" viewBox="-75 -75 150 150">
-							<circle class="bx--loading__stroke" cx="0" cy="0" r="37.5" />
-						</svg>
-					</div>
-				</td>
-			</tr>
-			<tr *ngIf="this.model.isEnd">
-				<td class="table_end-indicator">
-					<h5>{{getEndOfDataText() | async}}</h5>
-					<button (click)="scrollToTop($event)" class="btn--secondary-sm">
-						{{getScrollTopText() | async}}
-					</button>
-				</td>
-			</tr>
-		</tfoot>
-	</table>
+			[striped]="striped"
+			[skeleton]="skeleton">
+			<thead
+				ibmTableHead
+				[sortable]="sortable"
+				(deselectAll)="onDeselectAll()"
+				(selectAll)="onSelectAll()"
+				(sort)="doSort($event)"
+				[checkboxHeaderLabel]="getCheckboxHeaderLabel()"
+				[filterTitle]="getFilterTitle()"
+				[model]="model"
+				[size]="size"
+				[selectAllCheckbox]="selectAllCheckbox"
+				[selectAllCheckboxSomeSelected]="selectAllCheckboxSomeSelected"
+				[showSelectionColumn]="showSelectionColumn"
+				[enableSingleSelect]="enableSingleSelect"
+				[skeleton]="skeleton"
+				[sortAscendingLabel]="sortAscendingLabel"
+				[sortDescendingLabel]="sortDescendingLabel"
+				[stickyHeader]="stickyHeader">
+			</thead>
+			<tbody
+				ibmTableBody
+				(deselectRow)="onSelectRow($event)"
+				(scroll)="onScroll($event)"
+				(selectRow)="onSelectRow($event)"
+				[checkboxRowLabel]="getCheckboxRowLabel()"
+				[enableSingleSelect]="enableSingleSelect"
+				(rowClick)="onRowClick($event)"
+				[expandButtonAriaLabel]="expandButtonAriaLabel"
+				[model]="model"
+				[size]="size"
+				[ngStyle]="{'overflow-y': 'scroll'}"
+				[selectionLabelColumn]="selectionLabelColumn"
+				[showSelectionColumn]="showSelectionColumn"
+				[skeleton]="skeleton"
+				*ngIf="!noData; else noDataTemplate">
+			</tbody>
+			<ng-template #noDataTemplate><ng-content></ng-content></ng-template>
+			<tfoot>
+				<ng-template
+					[ngTemplateOutlet]="footerTemplate">
+				</ng-template>
+				<tr *ngIf="this.model.isLoading">
+					<td class="table_loading-indicator">
+						<div class="bx--loading bx--loading--small">
+							<svg class="bx--loading__svg" viewBox="-75 -75 150 150">
+								<circle class="bx--loading__stroke" cx="0" cy="0" r="37.5" />
+							</svg>
+						</div>
+					</td>
+				</tr>
+				<tr *ngIf="this.model.isEnd">
+					<td class="table_end-indicator">
+						<h5>{{getEndOfDataText() | async}}</h5>
+						<button (click)="scrollToTop($event)" class="btn--secondary-sm">
+							{{getScrollTopText() | async}}
+						</button>
+					</td>
+				</tr>
+			</tfoot>
+		</table>
+	</div>
 	`
 })
 export class Table implements AfterViewInit, OnDestroy {
```

Comment:
Where best fits in class, add @HostBinding as seen below. Run npm run lint:fix to ensure the order is corect.

```suggestion
export class Table implements AfterViewInit, OnDestroy {
	@HostBinding("class.bx--data-table-content") tableContent = true;
```